ui/app/adapters/node.js

```diff
@@ -61,4 +61,11 @@ export default class NodeAdapter extends Watchable {
       },
     });
   }
+
+  addMeta(node, newMeta) {
```

Comment:
`nit`: first param should probably just be `nodeId`. I was confused when we were passing `this` via the model.